app/relay.config.js

```diff
@@ -3,7 +3,8 @@ module.exports = {
   language: 'typescript',
   artifactDirectory: './__generated__',
   customScalars: {
-    Datetime: 'String'
+    Datetime: 'String',
+    JSON: 'any'
```

Comment:
I seem to recall there were reasons why this couldn't be a useful type

Reply:
It's not really useful, but by default it's defined as an `unknown`, which we were casting as an `any`